plutus-metatheory/src/Untyped/CEKWithCost.lagda.md

```diff
@@ -22,14 +22,15 @@ open import Untyped using (_⊢)
 open _⊢
 open import Utils using (Writer;_>>_;_>>=_;return;maybe;Either;inj₁;inj₂;RuntimeError;gasError;either;_∔_≣_)
 open Writer
-open import RawU using (tmCon)
+open import RawU using (tmCon;TmCon)
```

Comment:
Is `TmCon` ever used?

Reply:
It was on a version, but it is not needed now.